osbuild/mounts.py

```diff
@@ -126,8 +126,21 @@ def __init__(self, args):
     # pylint: disable=no-self-use
     @abc.abstractmethod
     def translate_options(self, options: Dict) -> List:
+        opts = []
         if options.get("readonly", False):
```

Comment:
You don't need the `False` argument to `.get("readonly")`. The default value when none is provided is `None` which is already falsey.